java/code/src/com/redhat/rhn/domain/channel/test/ChannelFactoryTest.java

```diff
@@ -112,12 +112,12 @@ public static Channel createBaseChannel(User user, String channelArchLabel) thro
     }
```

Comment:
Same comment as I had on the "add" PR.
This change only formatting. It would be better to have this in an extra commit.